### PR TITLE
Add natural=scree to landuse

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/vector-datasource/issues/655

@nvkelso, @zerebubuth could you review please?

### CODE REVIEW

**Comment by zerebubuth** [comment]
:+1:

File: spreadsheets/kind/landuse.csv [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
-*,*,*,*,*,*,*,*,*,*,groyne,*,"${GREATEST(LEAST(zoom, 16), 9)}",groyne
-*,*,*,*,*,*,*,*,*,*,dike,*,"${GREATEST(LEAST(zoom, 16), 9)}",dike
-*,*,*,*,*,*,*,*,*,*,cutline,*,"${GREATEST(LEAST(zoom, 16), 9)}",cutline
-*,*,*,*,*,*,*,*,*,*,*,plant,"${GREATEST(LEAST(zoom, 16), 9)}",plant
-*,*,*,*,*,*,*,*,*,*,*,generator,"${GREATEST(LEAST(zoom, 16), 9)}",generator
-*,*,*,*,*,*,*,*,*,*,*,substation,"${GREATEST(LEAST(zoom, 16), 9)}",substation
-*,*,*,*,*,*,*,*,*,*,*,station,"${GREATEST(LEAST(zoom, 16), 9)}",substation
-*,*,*,*,*,*,*,*,*,*,*,sub_station,"${GREATEST(LEAST(zoom, 16), 9)}",substation
-*,*,*,*,*,*,*,*,*,*,*,*,*,landuse
+boundary,leisure,landuse,natural,highway,amenity,aeroway,natural,tags->zoo,tags->attraction,tourism,man_made,power,min_zoom,kind
```
Comment (nvkelso) [inline spreadsheets/kind/landuse.csv]: Re-use the existing `natural` column?

Comment (rmarianski) [inline spreadsheets/kind/landuse.csv]: Nice catch! Updated in d836fb3.

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**